openpype/settings/defaults/project_anatomy/imageio.json

```diff
@@ -78,6 +78,10 @@
                         {
                             "name": "colorspace",
                             "value": "linear"
+                        },
+                        {
```

Comment:
Why is this in default if it does not have schemas and why it has value string `"True"`?

Reply:
Oh, it's modifiable dictionary...